### PR TITLE
Story/historic model graph/264

### PR DESCRIPTION
**Pull request by Kyubinhan**
- New weather data graph using d3 library
- Display historic model percentile values in a filled area fashion
- Address the case where `window.Keycloak` can be undefined
- Change the page title to MoreCast

### CODE REVIEW

File: src/features/auth/keycloak.ts [hunk 1]
```diff
@@ -8,10 +8,12 @@ export const kcInitOption: KeycloakInitOptions = {
...
-  realm: KC_REALM,
-  clientId: KC_CLIENT
-})
+const instance = window.Keycloak
+  ? window.Keycloak<'native'>({
+      url: KC_AUTH_URL,
+      realm: KC_REALM,
+      clientId: KC_CLIENT
+    })
+  : null
```
Comment (andrea-williams) [inline src/features/auth/keycloak.ts:17]: what effect does this change have?

Comment (Kyubinhan) [reply src/features/auth/keycloak.ts:17]: It prevents our app from crashing when`https://sso-dev.pathfinder.gov.bc.ca/auth/js/keycloak.js` is not available like last time when the platform service was upgrading their SSO

Comment (andrea-williams) [reply src/features/auth/keycloak.ts:17]: ahhhhhh nice. Smart!

File: src/features/fireWeather/components/TempRHGraph.tsx [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+        .attr('r', 1)
+      svg
+        .selectAll('.modelRHDot')
+        .data(modelValues)
+        .enter()
+        .append('circle')
+        .attr('class', 'modelRHDot')
+        .attr('cx', d => xScale(d.date))
+        .attr('cy', d => yRHScale(d.modelRH))
+        .attr('r', 1)
```
Comment (andrea-williams) [inline src/features/fireWeather/components/TempRHGraph.tsx:180]: nice work. I can imagine this required a lot of patience to get working correctly

**Pull request merged**